Source/CarthageKit/Project.swift

```diff
@@ -1034,14 +1034,17 @@ public final class Project { // swiftlint:disable:this type_body_length
 				return SignalProducer(dependencies)
 					.flatMap(.concurrent(limit: 4)) { dependency, version -> SignalProducer<(Dependency, PinnedVersion), CarthageError> in
 						switch dependency {
-						case .git, .gitHub:
+						case .git where options.build, .gitHub where !options.build:
```

Comment:
This may be a misunderstanding, but with installing binaries in the build process I assumed rather than check for no-build ahead of time on:
https://github.com/Carthage/Carthage/pull/2478/files#diff-46708ead17dbc4a66dd1b8eb0e73b9a9L39

I assumed this check could instead be done here? Otherwise, perhaps installBinaries may not be appropriate in the build step?

🤦‍♂️
Now that I'm writing this, there may have been another location to do this the whole time.